models/dex/dex_info.sql

```diff
@@ -94,4 +94,5 @@ FROM (VALUES
     , ('carbonhood', 'Carbonhood', 'Direct', 'carbonhood')
     , ('immortalx', 'ImmortalX', 'Direct', 'immortalx')
     , ('wardenswap', 'WardenSwap', 'Direct', 'wardenswap')
+    , ('pharaoh', 'Pharaoh', 'Direct', 'pharaoh')
```

Comment:
I just added pharaoh in to dex.info,
@discochuck you can edit this line if the information is wrong.